pmpro-approvals.php

```diff
@@ -514,6 +517,84 @@ public static function pmpro_approvals_user_row_actions($actions, $user) {
 		return $actions;
 	}
 
+	/**
+	 * Add Approvals status to Account Page.
+	 */
+	public static function pmpro_approvals_user_status_for_account(){
+
+			$approval_status = PMPro_Approvals::getUserApprovalStatus();
+
+			printf( __( '<li><strong>Status:</strong> %s</li>', 'pmpro-approvals'), $approval_status );

```

Comment:
Could rewrite this to keep the html out of the string.

    '<li><strong>' . __('Status', 'pmpro-approvals') . ':</strong> ' . $approval_status . '</li>'